app/controllers/classifier.coffee

```diff
@@ -90,6 +93,8 @@ class Classifier extends Controller
     @subjectViewer.setClassification @classification
     @animalSelector.setClassification @classification
 
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,subject.zooniverseId,'view',''
```

Comment:
What is the difference between selecting a subject and viewing it? presumably once selected it will always be rendered on screen?